pkg/tfbridge/provider.go

```diff
@@ -523,6 +523,10 @@ func (p *Provider) CheckConfig(ctx context.Context, req *pulumirpc.CheckRequest)
 		}
 	}
 
+	if err := p.typeCheckConfig(ctx, urn, news); err != nil {
+		return err, nil
+	}
```

Comment:
Let's call this `check` instead of `err`. By Go convention, `err` implies the type `error`.
```suggestion
	if check := p.typeCheckConfig(ctx, urn, news); check != nil {
		return check, nil
	}
```